dist/source/rodash.d.bs

```diff
@@ -307,36 +346,61 @@ end function
 ' * @param {Function} validator - A function used to validate the output value matches what you expected.
 ' * @return {Dynamic} The result of the drill down process
 ' */
-function get(aa as object, keyPath as string, fallback = Invalid as dynamic, validator = rodash_isNotInvalid as Function) as dynamic
+function get(aa as object, keyPath as string, fallback = Invalid as dynamic, validator = isNotInvalid as Function) as dynamic
```

Comment:
This is the fix for #6 . Notice we changed `validator = rodash_isNotInvalid` to `validator = isNotInvalid`